### PR TITLE
add method addMarkerViews with only one invalidate

### PR DESCRIPTION
**Pull request by SylvainHocq**
The profiling tools  allow me to see that invalidate method takes a long time .

My new  method addMarkerViews will make only one invalidate call.
That improve display of MarkerView on the map.
Indeed, you can see a some benchmark between the two methods :
(Time is write in millisecond, among  is the number of markers displayed )

With addMarker:
  amount:10, time:407
  amount:100, time:2824
  amount:500, time:19389
  amount:1000, time:74159

With addMarkerViews:
  amount:10, time:71
  amount:100, time:108
  amount:500, time:554
  amount:1000, time:775

### CODE REVIEW

**Comment by 13kz** [comment]
Seems great. I'm will check this out.

**Comment by tobrun** [comment]
Closes #5530, Thank you for contributing! will review soon

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/annotation/BulkMarkerActivity.java [hunk 1]
```diff
@@ -125,15 +125,24 @@ private void showViewMarkers(int amount) {
...
+            markerOptionsList.add(markerOptions);
         }
+        long t1 = System.currentTimeMillis();
+//        for (MarkerViewOptions markerViewOptions : markerOptionsList) {
+//            mapboxMap.addMarker(markerViewOptions);
+//        }
+        mapboxMap.addMarkerViews(markerOptionsList);
+        long t2 = System.currentTimeMillis();
+        Log.d("time", "amount:"+amount+", time:"+(t2-t1));
     }
```
Comment (tobrun) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/annotation/BulkMarkerActivity.java]: Could you remove the commented out code and profiling code?

**Comment by tobrun** [comment]
Could you rebase this onto master and possibly apply the nit comment? thanks for the contribution!

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/annotation/BulkMarkerActivity.java (hunk 1 above)
Comment (SylvainHocq) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/annotation/BulkMarkerActivity.java]: ok

**Comment by SylvainHocq** [comment]
@tobrun i have clean commit and merge the last master

**Pull request merged**